data/mods/DinoMod/monsters/juvenile.json

```diff
@@ -182,12 +195,40 @@
     "upgrades": { "age_grow": 80, "into": "mon_utahraptor" },
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "CATFOOD", "GUILT", "NO_BREED", "CANPLAY" ]
   },
+  {
+    "id": "mon_amargasaurus_juvenile",
+    "type": "MONSTER",
+    "name": "brown sauropod juvenile",
+    "copy-from": "mon_gallimimus_juvenile",
+    "looks_like": "mon_apatosaurus",
+    "description": "A huge four-legged plant-eating juvenile sauropod with a long neck and tail, it could be from a number of different species.",
```

Comment:
```suggestion
    "description": "A huge, four-legged, plant-eating juvenile sauropod with a long neck and tail, it could be from a number of different species.",
```